R/data.R

```diff
@@ -1032,7 +1053,7 @@
 #'   \item{high_emp}{}
 #'   \item{low_emp}{}
 #' }
-#' @source Bureau of Labor Statistics \url{https://www.bls.gov/oes/current/oes254021.htm#(1)}
+#' @source Bureau of Labor Statistics \url{https://www.bls.gov/oes/current/oes254022.htm}
```

Comment:
Should we remove the `\url{}` here too?

Reply:
It's weird. during the rhub checks, certain platforms spit out a warning when the url is a redirect url instead of native url. I only fixed the most pressing ones, but there are a BUNCH that still need to resolved.